### PR TITLE
Added support for serverID.

### PR DESCRIPTION
**Pull request by supimfuzzy**
I did change the description of ``g_hStatsWebsite``, ``g_hStatsTop`` and ``g_hDatabaseServerId`` to provide more info about using the serverid for server specific stats.

I am also not sure if ``g_hStatsTop`` supports the {SERVER} tag but if you could add it that would be great. It's needed because the top list will default to serverID 1.

The web interface is now server specific. This requires the serverid variable in the URL or else it will default to 1.

### CODE REVIEW

**Comment by splewis** [comment]
Maybe I'm wrong, but by disallowing severid=0 won't this break web stats entirely for lots of users unless they manually update all the entries in their tables?

**Comment by supimfuzzy** [comment]
Hm. You are right. Ill change up the queries.

**Comment by supimfuzzy** [comment]
It's not technically disallowed, if serverid URL parameter is set it would use the one defined if not it would default to 1. I have changed it to 0.

File: web/readme.md [hunk 1]
```diff
@@ -4,24 +4,27 @@ Multi 1v1 Arena Web Stats Interface
...
 If you are using the included ``multi1v1_online_stats_viewer`` plugin, you should set the ``sm_multi1v1_stats_url``  to something like:
-``http://www.domain.com/stats/redirect.php?id={USER}&server_id={SERVER}``. 
+``http://www.domain.com/stats/redirect.php?id={USER}&serverid={SERVER}`` and ``sm_multi1v1_top_url`` to something like: ``http://www.domain.com/stats/index.php?serverid={SERVER}``. 
 
-This cvar should be set in ``cfg/multi1v1/multi1v1_online_stats_viewer.cfg``.
+These cvars should be set in ``cfg/multi1v1/multi1v1_online_stats_viewer.cfg``.
 
-###Web Interface
-This web interface was a quick job that I randomly took up while I was learning PHP. I will try and keep it updated when I can, but no promises. If you have any issues setting it up, feel free to email me at clayton@hiddenempire.org
+**Note**: Don't forget that if you have multiple servers with different IDs you have to set the URL with the serverid in the plugin config or else it will only the stats for serverID 1.
```
Comment (splewis) [inline web/readme.md]: Should say serverID = 0 now

**Pull request merged**